tests/conftest.py

```diff
@@ -11,9 +11,13 @@
 from uuid import uuid4
 
 import pytest
-import trustme
-from cryptography.hazmat.backends import default_backend
-from cryptography.hazmat.primitives import serialization
+
+try:
+    import trustme
+    from cryptography.hazmat.backends import default_backend
+    from cryptography.hazmat.primitives import serialization
+except ImportError:  # pragma: nocover
+    trustme = None  # type: ignore
```

Comment:
I've done this to avoid mypy complaining about incompatible type of `trustme`. If you prefer another approach, just let me know and I'll update it.

Reply:
Can we follow this way: https://adamj.eu/tech/2021/12/29/python-type-hints-optional-imports, please?

Reply:
Sure, done that now.